v1.0/json-generator/consts.py

```diff
@@ -23,16 +23,16 @@ def __init__(self, genome=GENOME, read_length=READ_LENGTH, with_ercc=False, umis
         self.read_length = read_length
         _genome = genome.lower()
         if _genome == 'hg38' or _genome == 'grch38':
-            self.bowtie_genome_ref_first_index = "/data/reddylab/Reference_Data/Genomes/hg38/GCA_000001405.15_GRCh38_no_alt_analysis_set.1.ebwt"
-            self.bowtie2_genome_ref_first_index = "/data/reddylab/Reference_Data/Genomes/hg38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.bowtie_index.1.bt2" #TODO: Add ERCC92 Bowtie2 option
-            self.star_genome_dir = "/data/reddylab/Reference_Data/Genomes/hg38/STAR_genome_sjdbOverhang_%d_novelSJDB" % (self.read_length-1)
-            self.encode_blacklist_bedfile = "/data/reddylab/Reference_Data/ENCODE/hg38.blacklist.v3.bed"
+            self.bowtie_genome_ref_first_index = "/data/reddylab/Reference_Data/IGVF/GRCh38/bowtie_GRCh38/IGVFFI0653VCGH_GRCh38.1.ebwt"
+            self.bowtie2_genome_ref_first_index = "/data/reddylab/Reference_Data/IGVF/GRCh38/bowtie2_GRCh38/IGVFFI0653VCGH_GRCh38.1.bt2" #TODO: Add ERCC92 Bowtie2 option
+            self.star_genome_dir = "/data/reddylab/Reference_Data/IGVF/GRCh38/STAR_GRCh38_genome/" % (self.read_length-1)
+            self.encode_blacklist_bedfile = "/data/reddylab/Reference_Data/IGVF/GRCh38/hg38.blacklist.v3.bed"
             self.genome_effective_size = "hs"
-            self.annotation_file = "/data/reddylab/Reference_Data/Gencode/v22/gencode.v22.annotation%s.gtf" % (".with_ercc92" if with_ercc else "")
-            self.rsem_dir = "/data/reddylab/Reference_Data/Genomes/hg38/RSEM_genome%s"% (".with_ercc92" if with_ercc else "")
-            self.genome_sizes_file = "/data/reddylab/Reference_Data/Genomes/hg38/GCA_000001405.15_GRCh38_no_alt_analysis_set%s.sizes" % (".with_ercc92" if with_ercc else "")
-            self.genome_fasta_files = ["/data/reddylab/Reference_Data/Genomes/hg38/GCA_000001405.15_GRCh38_no_alt_analysis_set%s.fna" % (".with_ercc92" if with_ercc else "")]
-            self.regions_bed_file = "/data/reddylab/Reference_Data/Genomes/hg38/GCA_000001405.15_GRCh38_no_alt_analysis_set%s.bed" % (".with_ercc92" if with_ercc else "")
+            self.annotation_file = "/data/reddylab/Reference_Data/IGVF/GRCh38/Gencode/v43/IGVFFI7217ZMJZ.gtf" % (".with_ercc92" if with_ercc else "")
+            self.rsem_dir = "/data/reddylab/Reference_Data/IGVF/GRCh38/RSEM_genome"% (".with_ercc92" if with_ercc else "")
+            self.genome_sizes_file = "/data/reddylab/Reference_Data/IGVF/GRCh38/GCA_000001405.15_GRCh38_no_alt_analysis_set%s.sizes" % (".with_ercc92" if with_ercc else "")
+            self.genome_fasta_files = ["/data/reddylab/Reference_Data/IGVF/GRCh38/fasta/IGVFFI0653VCGH.fasta" % (".with_ercc92" if with_ercc else "")]
```

Comment:
All of these references do not exists when the RNA-seq samples have ERCC92 spike-ins. Please add those to fix the issue